aiidalab/registry/web.py

```diff
@@ -121,7 +122,6 @@ def build(
                 api_path=base_path / api_path,
                 apps_index=apps_index,
                 apps_data=apps_data,
-                scan_app_repository=parse_app_repo,
```

Comment:
This argument was completely unused in `build_api_v1`, perhaps copy-paste error or some historical remnant?